content/guides/core-concepts/cypress-app.md

```diff
@@ -6,29 +6,109 @@ title: Cypress App
+- How to debug tests using the built-in features of the Cypress app
 
 </Alert>
 
 ## Overview
 
-Cypress runs tests in a unique interactive application that allows you to see
-commands as they execute while also viewing the Application or Component Under
-Test.
+The Cypress desktop app has powerful features for creating, configuring,
+browsing, and most importantly running your tests. We'll touch on a few of the
+different functions before moving on to the Test Runner, the heart of the
+Cypress testing experience.
 
-<DocsImage src="/img/guides/cypress-app/gui-diagram.png" alt="Cypress App"></DocsImage>
+<Alert type="info">
+
+<strong class="alert-header">Dashboard Integration</strong>
+
+When you configure Cypress to record tests to the
+[Dashboard](/guides/dashboard/introduction) you'll see data from your latest
+recorded runs in the Cypress app, increasing visibility into your test history.
+If you're not sure whether you're logged in, you can always check using the
+profile control in the top right of the window.
+
+The data used by these features is pulled from the Dashboard, so they will only
+reflect test results that were
+[recorded](/guides/dashboard/projects#Set-up-a-project-to-record).
+
+</Alert>
+
+### The Launchpad
+
+<DocsImage src="/img/guides/core-concepts/cypress-app/the-launchpad.png" alt="The Launchpad"></DocsImage>
+
+The Launchpad is your portal to Cypress, helping with onboarding, choosing a
+testing type and launching a browser.
+
+Once you get comfortable with Cypress you might find you don't need the
+Launchpad any more. In this case
+[you can run `cypress open` with the `--browser` and `--component` or `--e2e` options](/guides/guides/command-line#cypress-open)
+to go straight to the Spec Explorer.
+
+### The Spec Explorer
+
+TODO Image with rest of columns
+<DocsImage src="/img/guides/core-concepts/cypress-app/spec-explorer.png" alt="The Spec Explorer"></DocsImage>
+
+On choosing your browser in the Launchpad, you'll be presented with a list of
+your specs, their names, locations, and data from your latest recorded runs.
+Here you can launch specs by clicking them, create new blank or example specs,
+or search specs by name (handy for large test suites).
+
+We've made the following spec metadata available directly in the Spec Explorer:
+
+#### Last updated
+
+This column shows when the code of this spec was last updated, using your local
+system's own metadata. This metadata will vary from machine to machine, as it
+reflects the state of the code as stored on that computer.
+
+#### Latest runs
+
+This column shows the last time the spec was run, and the last four returned
+statuses. This metadata comes from the Dashboard (see [note above](#Overview))
+so should be the same everywhere.
+
+#### Average duration
+
+This column shows the average time taken to run the entire spec file, over the
+last four runs. Again, this metadata comes from the Dashboard.
```

Comment:
These sections should focus more on the value that the data provides.

Reply:
Value narrative notes:
- **Last updated** showing the change history of your tests so you can quickly find the most relevant specs as you’re browsing your test suite
- **Latest runs** so you can monitor, run, and fix tests locally within CI workflows, and then further dig in to test results by clicking through to the Cypress Dashboard
- **Average duration** of your specs so that you can quickly identify and take action to improve the performance of your long-running specs